example/data-migration1-tf/README.md

```diff
@@ -0,0 +1,7 @@
+# Data migration deployment module
+
+## How to get value for `provider_kms_key_id` Terraform variable
```

Comment:
I don't really like this option, but there's no other way to easily get the provider KMS key ID to use for this module. This module has to deploy **before** the cumulus module, so it can't be retrieved as an output from the cumulus module

Reply:
I'm fine with this solution.